test/temperature.js

```diff
@@ -311,6 +311,62 @@ exports["Temperature -- ANALOG"] = {
       K: 296
     }),
   },
+
+  GROOVE: {
+    setUp: function(done) {
+      this.temperature = new Temperature({
+        controller: "GROVE",
+        pin: "A0",
+        freq: 100,
+        board: this.board
+      });
+
+      done();
+    },
+    shape: testShape,
+    aref: makeTestAnalogConversion({
```

Comment:
can someone verify that these two controllers should see no change when aref is different?